base/src/expressions/lexer/test/test_common.rs

```diff
@@ -411,6 +411,20 @@ fn test_name_r1c1p() {
     assert_eq!(lx.next_token(), EOF);
 }
 
+#[test]
+fn test_reference_r1c1_error() {
```

Comment:
Halfway through the code I changed my mind and I don't call test functions these days with names starting with `test`. I feel it is a little bit too much.

Reply:
I think that makes sense with the `#[test]` macro above it.  I'll leave the `test_` prefix off moving forward.  I can quickly remove the prefix in the file altogether if you prefer?

Reply:
It's ok in this file. I was just leting you know of my preferences. It's not a big deal.